depends/config.site.in

```diff
@@ -55,9 +55,12 @@ export PKG_CONFIG_PATH=$depends_prefix/share/pkgconfig
 CPPFLAGS="-I$depends_prefix/include/ $CPPFLAGS"
 LDFLAGS="-L$depends_prefix/lib $LDFLAGS"
 
-CC="@CC@"
-CXX="@CXX@"
-OBJC="${CC}"
```

Comment:
at best 14036 should be marked partial.

This doesn't actually backport the logic of running `unit tests --with-sanitizers=undefined` in CI